### PR TITLE
Adds Contributing document, bug and feature request templates

### PR DESCRIPTION
**Pull request by Golodhros**
### Summary of Changes
Adds Contributing document with instructions about how to get started contributing to Amundsen
Adds a bug and feature request templates

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely.
- [x] PR includes a summary of changes.
- [x] My commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+## Screenshots (if appropriate)
+
+## Context
+<!--- How has this issue affected you? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the bug in -->
+* Amunsen version used:
+* Browser Name and version:
```
Comment (feng-tao) [inline .github/ISSUE_TEMPLATE/bug-report.md:39]: Normally the issue is on the backend or ingestion or external connection. I felt the browser section as well as OS may not be very relevant.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+title: Feature Proposal
+labels: enhancement
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the feature request or improvement in the Title above -->
+<!--- Look through existing open and closed feature proposals to see if someone has asked for the feature before -->
+
+## Expected Behavior
```
Comment (feng-tao) [inline .github/ISSUE_TEMPLATE/feature-request.md:13]: maybe add a usecase section as well as which service/ingestion etl is involved.

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+<!--- Tell us how it should work -->
+
+## Difference from Current Behavior
+<!--- Explain the difference from current behavior -->
+
+## Possible Implementation
```
Comment (feng-tao) [inline .github/ISSUE_TEMPLATE/feature-request.md:19]: Also add a section includes adding new model / changing existing models etc?
also Migration Plan and Compatibility ?
also potential Rejected Alternatives ?

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+---
+
+<!--- Provide a general summary of the feature request or improvement in the Title above -->
+<!--- Look through existing open and closed feature proposals to see if someone has asked for the feature before -->
+
+## Expected Behavior
+<!--- Tell us how it should work -->
+
+## Difference from Current Behavior
```
Comment (feng-tao) [inline .github/ISSUE_TEMPLATE/feature-request.md:16]: if it is feature, normally the request just doesn't exist today? not sure if we need a difference section as I assume it is applicable to bug?

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (Golodhros) [reply .github/ISSUE_TEMPLATE/bug-report.md:39]: Makes sense! Removing those.

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 4 above)
Comment (Golodhros) [reply .github/ISSUE_TEMPLATE/feature-request.md:16]: True, removing this one

File: .github/ISSUE_TEMPLATE/feature-request.md (hunk 3 above)
Comment (Golodhros) [reply .github/ISSUE_TEMPLATE/feature-request.md:19]: All of those questions are really valid, however we could overwhelm a bit our users if we ask all that. I see the feature request issue as a simple "idea gathering" tool. 

I think these are questions for a design document, something we would probably would need to drive. Let me add them into a section of the contributing guide.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (feng-tao) [reply .github/ISSUE_TEMPLATE/bug-report.md:39]: hey @Golodhros , the Browser and Operating System are still there?

**Pull request merged**